src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -0,0 +1,137 @@
+    <command source="client" code="0x00" name="SetpointRaiseLower" optional="false" cli="zcl tstat set">
+      <description>
+        Command description for SetpointRaiseLower
+      </description>
+      <arg name="mode" type="SetpointAdjustMode"/>
+      <arg name="amount" type="INT8S"/>
+    </command>
+    <command source="client" code="0x01" name="SetWeeklySchedule" optional="false" cli="zcl tstat sws" introducedIn="ha-1.2-05-3520-29">
+      <description>
+        Command description for SetWeeklySchedule
+      </description>
+      <arg name="numberOfTransitionsForSequence" type="ENUM8"/>
+      <arg name="dayOfWeekForSequence" type="DayOfWeek"/>
+      <arg name="modeForSequence" type="ModeForSequence"/>
+      <arg name="payload" type="INT8U" array="true"/>
+    </command>
+    <command source="client" code="0x02" name="GetWeeklySchedule" optional="false" cli="zcl tstat gws" introducedIn="ha-1.2-05-3520-29">
+      <description>
+        Command description for GetWeeklySchedule
+      </description>
+      <arg name="daysToReturn" type="DayOfWeek"/>
+      <arg name="modeToReturn" type="ModeForSequence"/>
+    </command>
+    <command source="client" code="0x03" name="ClearWeeklySchedule" optional="false" cli="zcl tstat cws" introducedIn="ha-1.2-05-3520-29">
+      <description>
+        The Clear Weekly Schedule command is used to clear the weekly schedule.
+      </description>
+    </command>
+    <command source="client" code="0x04" name="GetRelayStatusLog" optional="false" cli="zcl tstat grs" introducedIn="ha-1.2-05-3520-29">
+      <description>
+        The Get Relay Status Log command is used to query the thermostat internal relay status log.
+      </description>
+    </command>
+    <command source="server" code="0x00" name="CurrentWeeklySchedule" optional="false" introducedIn="ha-1.2-05-3520-29">
+      <description>
+        The Current Weekly Schedule Command is sent from the server in response to the Get Weekly Schedule Command.
+      </description>
+      <arg name="numberOfTransitionsForSequence" type="ENUM8"/>
+      <arg name="dayOfWeekForSequence" type="DayOfWeek"/>
+      <arg name="modeForSequence" type="ModeForSequence"/>
+      <arg name="payload" type="INT8U" array="true"/>
+    </command>
+    <command source="server" code="0x01" name="RelayStatusLog" optional="false" introducedIn="ha-1.2-05-3520-29">
+      <description>
+        This command is sent from the thermostat cluster server in response to the Get  Relay Status Log.
+      </description>
+      <arg name="timeOfDay" type="INT16U"/>
+      <arg name="relayStatus" type="BITMAP16"/>
+      <arg name="localTemperature" type="INT16S"/>
+      <arg name="humidityInPercentage" type="INT8U"/>
+      <arg name="setpoint" type="INT16S"/>
+      <arg name="unreadEntries" type="INT16U"/>
+    </command>
+  </cluster>
+
+  <bitmap name="ThermostatFeature" type="BITMAP32">
+    <field name="Heating" mask="0x1"/>
```

Comment:
Needs a <cluster> element here so the right strongly typed enum will be generated for use with BitFlags.

Reply:
Fixed